app/assets/stylesheets/mobile/conversations.scss

```diff
@@ -1,10 +1,11 @@
+.conversations-title { margin: 0 0 20px 0; }
 .conversation {
-
   .conversation_participants {
     padding: 1rem 1.2rem;
 
     h3 {
-      margin: 0px;
+      margin: 0;
+      display: inline;
```

Comment:
I guess you wanted to add `display: inline` to `.conversations-title h3` instead.